website/docs/components/form/primitives/partials/code/how-to-use.md

```diff
@@ -30,6 +30,8 @@ If the label needs to contain more than just text, it’s possible to pass struc
 
 !!! Warning
 
+**Accessibility consideration**
```

Comment:
[question] should this be a "Accessibility alert", since it's a "warning"? or the "warning" should be an "info"?
```suggestion
**Accessibility alert**
```